ckanext/benap/templates/package/snippets/nap_checked_pill.html

```diff
@@ -1,4 +1,5 @@
 <span class="dataset-checked pull-right label label-checked-inverse ">
+    <a href=""
```

Comment:
moet er hier niet nog iets als href, nu is het gewoon een lege string?

Reply:
oei, rommel, blijven staan precies.